metadata-ingestion/src/datahub/ingestion/source/powerbi/config.py

```diff
@@ -195,6 +211,8 @@ def workspace_id_backward_compatibility(cls, values: Dict) -> Dict:
         return values
 
 
-class PowerBiDashboardSourceConfig(PowerBiAPIConfig):
+class PowerBiDashboardSourceConfig(
+    PowerBiAPIConfig,
```

Comment:
why do we have both PowerBiAPIConfig and PowerBiDashboardSourceConfig?

Reply:
deleted the PowerBiAPIConfig